Levey jennings report

nizwant opened the pull request.

Fersoil commented on R/generate_report.R:
```diff
@@ -101,26 +101,59 @@ generate_plate_report <-
+#' )
+#' note <- "This is a Levey-Jennings report.\n**Author**: Jane Doe \n**Tester**: John Doe"
+#'
+# ' generate_levey_jennings_report(
```
here remove the whitespace between # and ' - this prevents roxygen2 from generating the example properly

Fersoil reviewed (changes requested):
also add `generate_levey_jennings_report` to _pkgdown.yaml

I got this error from running the example:
```
Quitting from lines 12-68 [param-check] (levey_jennings_report_template.Rmd)
Error in `as.POSIXct.numeric()`:
! argument 'origin' musi zostać dostarczony
Backtrace:
 1. base::difftime(date_of_last_plate, date_of_first_plate, units = "days")
 3. base::as.POSIXct.numeric(time1)
 ```

nizwant commented:
> are you able to reproduce it?

While i was not able to reproduce it, I introduced changes that should address it. What is wierd is the fact that we deal with dates ealier in the pipeline, at the level of reading raw data. I also added `generate_levey_jennings_report` to vignette.

Fersoil reviewed (approved):
Now is looking good
